src/core/Akka.TestKit/ActorCellKeepingSynchronizationContext.cs

```diff
@@ -41,23 +41,25 @@ public ActorCellKeepingSynchronizationContext(ActorCell cell)
         /// <param name="state">TBD</param>
         public override void Post(SendOrPostCallback d, object state)
         {
-            ThreadPool.QueueUserWorkItem(_ =>
+            ThreadPool.QueueUserWorkItem(s =>
             {
+                var t = ((SendOrPostCallback, object, ActorCellKeepingSynchronizationContext, ActorCell))s;
```

Comment:
This only affects the `TestKit` so I think we should be ok

Reply:
I don't know what's faster here `ValueTuple` or `System.Tuple`, wanted only remove the scope. 

It does not matter here, but when a scope is used with ThreadPool, 
the context will not be GC until the action runs, it can lead to a self-reference in some scenarios.